containers/session-containers/skaha-desktop/apply-version.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 . VERSION && echo "tags: $TAGS"
 for t in $TAGS; do
-   docker image tag images.canfar.net/skaha/desktop:latest images.canfar.net/skaha/desktop:$t
```

Comment:
This should probably remain as images, not images-rc.  Or you could make it do both.

Reply:
Right. Changed back to images.